build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.5.0" % "test",
   "junit" % "junit" % "4.13.2" % "test",
   "com.github.sbt" % "junit-interface" % "0.13.3" % "test",
-  "org.apache.logging.log4j" % "log4j-core" % "2.19.0" % "test",
+  "org.apache.logging.log4j" % "log4j-core" % "2.20.0" % "test",
```

Comment:
logging dependencies should no longer be needed and can be removed.

Daffodil 3.5.0 dropped support for log4j and switched to slf4j. Additionally, we added a custom sl4fj logger implementation, which is a dependency of daffodil-tdml-processor used to capture and expect logs.

So API users might need to add a slf4j implementation if they want logging, but schema projects should no longer need it.